src/components/Avatar/index.js

```diff
@@ -1,22 +1,76 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { cx } from "emotion"
-import { imgWrapper, wrapper } from "./styles"
+import {
+  imgWrapper,
+  wrapper,
+  marginBottom,
+  avatarMobile,
+  avatarImage,
+  letters
+} from "./styles"
+import { UserType } from "../../type"
 
-Avatar.propTypes = {
+// Renders the Avatar with Image
+const AvatarImage = ({ size = "medium", user, className }) => {
+  return (
+    <div className={cx(wrapper, { large: size === "large" }, className)}>
+      <div className={imgWrapper}>
+        <img src={user.image} alt="avatar" />
+      </div>
+    </div>
+  )
+}
+
+AvatarImage.propTypes = {
   className: PropTypes.string,
   size: PropTypes.oneOf(["medium", "large"]),
-  src: PropTypes.string
+  user: UserType
 }
 
-function Avatar({ size = "medium", src, className }) {
+// Renders avatar with the letters
+const AvatarText = ({ size = "medium", user, className }) => {
+  const firstNameLetter = user.firstName.slice(0, 1)
+  const lastNameLetter = user.lastName.slice(0, 1)
```

Comment:
It would be nice to capitalise those letters before we render them

Reply:
It happens with css